packages/react-native-editor/src/index.js

```diff
@@ -56,6 +57,11 @@ const setupInitHooks = () => {
 		'core/react-native-editor',
 		( props ) => {
 			setupLocale( props.locale, props.translations );
+
+			const capabilities = props.capabilities ?? {};
+			if ( capabilities.reusableBlock !== true ) {
+				unregisterBlockType( 'core/block' );
```

Comment:
I initially considered hiding the block type as we do for other cases like Audio blocks ([code reference](https://github.com/WordPress/gutenberg/blob/bc5f0fe1269366d3f95fdfaf2d410bdb8a08a277/packages/react-native-editor/src/index.js#L112-L116)) but I finally decided to unregister it because the reusable block should be unsupported for self-hosted sites.

Reply:
Understood! 🙇🏾 Thanks for sharing. That case with the Audio block will be refactored as the Audio block will be available on all site types but some of its upload functionality will be restricted by site type.

Reply:
This may not be related but an error I discovered while testing the main apps PR is that the unsupported reusable block does not have a name. When a block is unregistered does it remove its name?